src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -473,6 +496,39 @@ func (c controller) postInstallConfigs(ctx context.Context) error {
 	return nil
 }
 
+func (c controller) getReadyOperators() ([]string, []models.MonitoredOperator, error) {
+	var readyOperators []string
+	operators, err := c.ic.GetClusterMonitoredOLMOperators(context.TODO(), c.ClusterID)
+	if err != nil {
+		return readyOperators, operators, err
+	}
+
+	for index := range operators {
+		handler := NewClusterServiceVersionHandler(c.kc, &operators[index], c.Status)
+		operatorStatus, _, err := handler.GetStatus()
+		if err != nil {
+			return readyOperators, operators, err
+		}
+		if utils.IsStatusSucceeded(operatorStatus) {
+			readyOperators = append(readyOperators, handler.GetName())
+		}
+	}
+	return readyOperators, operators, nil
+}
+
+func (c controller) waitForCSVsBeReady() bool {
```

Comment:
what means waitForCSVsBeReady ? does it mean that we wait till OLM operator is available already? 
I think i just missed something cause i don't know enough regarding the flow.
But doesn't it means that we wait for OLM operator to be ready twice ? or maybe it is another status that we check here? Just trying to understand

Reply:
You are right, we wait twice. First we wait to be sure the subscription is properly applied, and so it makes sense to apply also the CR of the OLM operator. Second time we wait that the CR is properly applied and CSV report proper state.

Reply:
I think it should be `waitForCSVBeCreated`